src/neural/opencl/network_opencl.cc

```diff
@@ -57,11 +57,14 @@ struct OpenCLWeights {
 
 class OpenCLComputation : public NetworkComputation {
  public:
-  OpenCLComputation(const OpenCL_Network& opencl_net,
-                    const OpenCLWeights& weights)
-      : opencl_net_(opencl_net), weights_(weights), policies_(), q_values_() {}
+  OpenCLComputation(OpenCL_Network& opencl_net, const OpenCLWeights& weights)
```

Comment:
Non-const reference parameters are not allowed. Replace with a pointer. (also in other functions)

Reply:
Done for the two constructor-calls with non-const ref. parements: OpenCLComputation and OpenCLBuffers.